basic_and_faq_usages/simple_artifact_store.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+from optuna.artifacts import download_artifact
+from optuna.artifacts import FileSystemArtifactStore
+from optuna.artifacts import get_all_artifact_meta
+from optuna.artifacts import upload_artifact
+import pandas as pd
+
+
+dataset_path = "demo-dataset.csv"
+fig_name = "result-trial.png"
+# NOTE: The uploaded artifacts can be viewed in Optuna Dashboard with the following command:
+# $ optuna-dashboard sqlite:///artifact-demo.db --artifact-dir ./save-artifact-here
+base_path = os.path.join("./save-artifact-here")
```

Comment:
It doesn't need to use `os.path.join()` since only one argument is being passed. How about updating this like following?

```suggestion
base_path = os.path.join(".", "save-artifact-here")
```

or

```suggestion
base_path = "./save-artifact-here"
```

Reply:
Thank you for the review!
I took the second option!